src/test/scala/com/gu/identity/play/CookieBuilderTest.scala

```diff
@@ -0,0 +1,69 @@
+package com.gu.identity.play
+
+import org.joda.time.format.ISODateTimeFormat
+import org.joda.time.{DateTime, Duration}
+import org.scalactic.Tolerance._
```

Comment:
Used there https://github.com/guardian/identity-play-auth/pull/3/files#diff-3d48a75af0e0879fd059283dcba52731R56
But never heard of scalatestplus, so maybe it does it as well ?